Package.swift

```diff
@@ -9,8 +9,15 @@ let package = Package(
         .library(name: "Sentry-Dynamic", type: .dynamic, targets: ["Sentry"])
     ],
     targets: [
+        .target( name: "SentrySwift",
+                 path: "Sources",
+                 sources: [
+                    "SentrySwift"
```

Comment:
`m`: I think we don't need to prefix that. We could rename the folder to `Swift` only. What do you think?
```suggestion
                    "Swift"
```